docs/getting-started/quickstart.rst

```diff
@@ -13,9 +13,11 @@ execute the following commands in the terminal:
 
 .. note::
 
-    On Windows, you have to open up the conda environment file and delete the
-    lines with ``sherpa`` and ``healpy``. Those are optional dependencies that
-    currently aren't available on Windows.
+    On Windows, the optional dependencies ``sherpa`` and ``healpy`` aren't available.
+
+.. note::
+
+    On Linux, the optional dependency ``sherpa`` isn't available.
```

Comment:
```suggestion
    On Linux, the optional dependency ``sherpa`` isn't installed by default.
    If required, it can be installed with `conda install -c https://cxc.cfa.harvard.edu/conda/sherpa -c conda-forge sherpa`
```